tests/test_discover_api.py

```diff
@@ -895,3 +899,21 @@ def test_count_null_user_consistency(self):
         assert len(data["data"]) == 1
         # Should now count '' user as Null, which is 0
         assert data["data"][0]["uniq_user"] == 0
+
+    def test_individual_measurement(self) -> None:
+        response = self.app.post(
+            "/query",
+            data=json.dumps(
+                {
+                    "dataset": "discover",
+                    "project": self.project_id,
+                    "selected_columns": ["event_id", "measurements[lcp]"],
```

Comment:
I would test `lcp.elementSize` too, to make sure the extra dots don't mess things up.

Reply:
Bump, but it's up to you.